exemplu2.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+    <head >
+    
+        
+       
+    <section style="width:100%; height:300px">
```

Comment:
This isn't right! The `<head>` section of a document cannot contain displayable elements. These should go in the document's `<body>`.